consensus/poet/core/sawtooth_poet/database/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+__all__ = [
```

Comment:
No, these are shim classes, to create the same behaviour as the when PoET was embedded in the validator.  The consensus engine sdk doesn't supply access to blocks, etc through the lens of a database, because that is an internal structure to the validator.

Reply:
Per discussion, this is good as is. Some consensus implementations may bring their own store / database implementations for consensus information. Probably not optimal to provide default database capabilities through the consensus SDK.